icons/favicon-dark-support.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="100%" height="100%" viewBox="0 0 400 400" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <style>
+    style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;"><style>
+    @media (prefers-color-scheme: dark) {
+    * {
+        fill: white;
+    }
+    }
+    </style>
+    <title>Evergreen Logo</title>
+    <g id="Evergreen-Reconfigure-Patch" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group-22" fill="#71CC97">
+            <path d="M98.3459261,8.56398024e-14 C65.5639507,119.288136 32.7819754,240.915254 4.28199012e-14,360.20339 C29.0338608,356.687078 56.197672,353.186441 86.6380777,348.508475 C79.105121,396.233522 86.6380777,351.874653 79.105121,400 L119.928301,400 C112.395344,348.508475 119.928301,400 112.395344,348.508475 C114.736914,348.508475 154.543598,355.525424 194.350282,362.542373 C182.642434,313.423729 140.49418,334.104351 98.3459261,277.968757 C121.761623,289.663673 161.568307,299.389831 182.642434,301.728814 C168.593016,252.610169 135.811041,280.30774 98.3459261,231.189096 C117.078483,240.545028 149.860459,247.932203 168.593016,252.610169 C154.543598,203.491525 131.127901,226.51113 98.3459261,184.409435 C114.736914,191.426384 140.49418,198.813559 154.543598,203.491525 C142.83575,154.372881 128.786332,172.71452 98.3459261,135.290791 C112.395344,142.30774 128.786332,147.355932 140.49418,152.033898 C135.031666,130.20798 131.608062,122.55807 126.656026,117.04381 C120.994325,110.739314 113.334729,107.226437 98.3459261,88.5111302 C107.712205,93.1890963 117.078483,98.2372881 126.444762,100.576271 C114.736914,58.4745763 124.103192,72.1382488 98.3459261,41.7314692 C105.370635,44.0704522 107.712205,46.779661 112.395344,49.1186441 L98.3459261,8.56398024e-14 Z" id="Path"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
one of the online svg viewers I put this into said it wasn't valid svg, and the other failed to visualize. I don't know much about svg but might be cautious